source/use-cases/sensitive-data-encryption.txt

```diff
@@ -0,0 +1,312 @@
+                "algorithm": "AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+            }
+            },
+            "fullName": {
+            "bsonType": "string",
+            "description": "must be a string and is required"
+            },
+            "patientInfo": {
+            "bsonType": "object",
+            "properties": {
+                "phone": {
+                "encrypt": {
+                    "bsonType": "int",
+                    "keyId": [
+                    {
+                        "$binary": {
+                        "base64": <keyId>,
+                        "subType": "04"
+                        }
+                    }
+                    ],
+                    "algorithm": "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic"
+                }
+                },
+                "provider": {
+                "bsonType": "string",
+                "description": "must be a string and is required"
+                }
+            }
+            },
+            "ssn": {
+            "encrypt": {
+                "bsonType": "int",
+                "keyId": [
+                {
+                    "$binary": {
+                    "base64": "<keyId>",
+                    "subType": "04"
+                    }
+                }
+                ],
+                "algorithm": "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic"
+            }
+            }
+        }
+    }
+
+
+C. Configure the MongoDB Client
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Donec sem felis, iaculis id pharetra sed, sollicitudin ut nisl.
+Suspendisse commodo sagittis ante in ullamcorper. Etiam viverra
+pellentesque suscipit. Pellentesque interdum egestas libero, ut varius
+velit congue vel.
+
+D. Perform Encrypted Read/Write Operations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Etiam hendrerit posuere justo, ac hendrerit lorem pellentesque sed.
+Fusce ullamcorper neque eu libero aliquam, sed placerat sapien cursus.
+Vestibulum maximus magna sed velit feugiat tempor.
+
+Ut accumsan pellentesque metus, at porta felis dapibus vulputate. Aenean
+malesuada interdum libero in feugiat. Fusce pharetra massa eget leo
+varius, et gravida massa tempus.
+
+Ut mattis eros elit, sed ultrices purus lacinia sed. Curabitur tincidunt
+tellus eget neque porta dictum ac vel orci. Vestibulum ante ipsum primis
+in faucibus orci luctus et ultrices posuere cubilia Curae.
+
+Praesent feugiat arcu non risus elementum eleifend. Nullam sed iaculis
+erat.
+
+E. Convert to a Remote Master Key
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Fusce porta lectus et enim vehicula malesuada. In porttitor ipsum eu
+dictum congue. Fusce pharetra sagittis lacus sed posuere. Donec porta
+posuere gravida. Quisque nisi massa, condimentum vitae ligula eget,
+fringilla malesuada quam. Mauris in semper nulla, non blandit tortor.
+Mauris consequat eget urna at eleifend. Suspendisse potenti. Fusce at
+justo malesuada, finibus felis non, dignissim nisl. Ut blandit nulla sit
+amet massa euismod vehicula.
```

Comment:
Maybe instead of Lorem ipsum just put TODO or something so someone can see at a glance what needs to be done still.

Reply:
I'd actually prefer it as is. Since mostly Chris Cho & I are working on this, I think it should be fine & is already clear what needs to be done. But if the team would prefer "to do" to the lorem ipseum, I'll change it.  @ccho-mongodb & @nlarew what do you think?

Reply:
Fine with me!